lib/bots/syncbot.ts

```diff
@@ -269,8 +269,8 @@ export class SyncBot extends ProcBot {
 				title: data.details.title,
 			},
 			source: {
-				message: 'duff',
-				thread: 'duff',
+				message: 'duff_SyncBot_createErrorComment_a',
```

Comment:
Just a comment currently, but maybe we should add in future something that allows the creation of a unique set of identifier messages, something that texts a text string then appends line number to it?

Reply:
Good shout.  Will add an issue.